common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3938,6 +3938,18 @@
     ],
     "sqlState" : "0A000"
   },
+  "NOT_SUPPORTED_CHANGE_PARENT_CHILD_COLUMN" : {
+    "message" : [
+      "ALTER TABLE ALTER/CHANGE COLUMN is not supported for changing <table>'s parent field <parentField> and child field <childField> in the same command."
+    ],
+    "sqlState" : "0A000"
+  },
+  "NOT_SUPPORTED_CHANGE_SAME_COLUMN" : {
```

Comment:
We can always use this error class, as changing the nested field also means changing its parent column.